src/pages/signin/ValidateCodeForm/BaseValidateCodeForm.js

```diff
@@ -89,15 +90,11 @@ function BaseValidateCodeForm(props) {
             return;
         }
 
-        if (!validateCode) {
-            return;
-        }
```

Comment:
We can add, I thought it will again create the same issue of linkSent but in particular this part it is relying on `prevIsVisible && !props.isVisible`. I will add it.